EMCAL/EMCALsim/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ include_directories(${AliRoot_SOURCE_DIR}/EMCAL/${MODULE})
 # Additional include folders in alphabetical order except ROOT
 include_directories(SYSTEM ${ROOT_INCLUDE_DIR})
 include_directories(${AliRoot_SOURCE_DIR}/EMCAL/EMCALbase
+                    ${AliRoot_SOURCE_DIR}/EMCAL/EMCALTriggerBase
```

Comment:
Eventually we need to move the constants file to EMCALutils, not sure about the dependency here. I would prefer to have the dependencies as lightweight as possible.